.github/workflows/cla.yml

```diff
@@ -0,0 +1,32 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
```

Comment:
I think we want to skip running the CLA Assistant when anyone from our organisation makes PRs

We can do that by making the job conditional (we could have extended the `if` for the step too, but it is already so long)

I think this could work, add it after `runs-on:`

```yaml
if: ${{ github.event.pull_request.author_association != 'MEMBER' || github.event.pull_request.author_association != 'OWNER' }}

Got this from looking at these docs for the event payload https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request (expand `pull_request`)

and https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
added 
`if: ${{ github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}`